lighthouse-core/gather/gather-runner.js

```diff
@@ -246,6 +272,9 @@ class GatherRunner {
     // Example: `DNS_FAILURE` is better than `NO_FCP`.
     if (networkError) return networkError;
 
+    // We want to error when the page is not of MIME type text/html
+    if (docTypeError) return docTypeError;
```

Comment:
I assumed that the order of importance of errors was interstitial -> network -> docType, should this be the case?